test/test_nested_tensor_functional.py

```diff
@@ -24,6 +24,24 @@ def test_addmm(self):
             [torch.rand(1, 4), torch.rand(1, 4), torch.rand(4, 4)]
         )
 
+    def test_contiguousity(self):
```

Comment:
This should be done for all functions where we expect contiguity. Also nit: it's spelled
[contiguity](https://en.wikipedia.org/wiki/Contiguity).

We can make this a follow-up for this PR, but should do so asap.